build.gradle

```diff
@@ -3,7 +3,13 @@ task monoBuild {
     dependsOn(':frontend:buildClientDev', ':frontend:copyFrontendFiles', ':backend:bootRun')
 }
 
+task buildDockerImage {
+    description = 'Build Docker image'
+    dependsOn(':frontend:buildClientDev', ':frontend:copyFrontendFiles', ':backend:dockerBuildImage')
+}
+
 tasks.getByPath(':backend:bootRun').mustRunAfter(':frontend:copyFrontendFiles')
+tasks.getByPath(':backend:dockerBuildImage').mustRunAfter(':frontend:copyFrontendFiles')
```

Comment:
Do we need this line if it's already in the `dependsOn` in line 8?

Reply:
Line 8 adds those tasks as dependencies, but does not specify any mutual ordering between them. And `mustRunAfter` alone does not imply execution dependency, the rule only applies if both tasks are scheduled. ([docs](https://docs.gradle.org/current/userguide/more_about_tasks.html))